src/main/java/com/commercetools/sync/sdk2/inventories/helpers/InventoryEntryIdentifier.java

```diff
@@ -0,0 +1,97 @@
+package com.commercetools.sync.sdk2.inventories.helpers;
+
+import static java.lang.String.format;
+
+import com.commercetools.api.models.channel.ChannelReference;
+import com.commercetools.api.models.channel.ChannelResourceIdentifier;
+import com.commercetools.api.models.inventory.InventoryEntry;
+import com.commercetools.api.models.inventory.InventoryEntryDraft;
+import java.util.Objects;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+/**
+ * This class provides as a container of the unique identifier of an {@link
+ * io.sphere.sdk.inventory.InventoryEntry} for the sync which is a combination of the SKU and the
```

Comment:
Please check the links in comments and update to sdk2.

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/931/commits/37c82ff9880900242f304c60a89bf06e4ede69fd